homework6/task1_counter/tests/test_instance_counter.py

```diff
@@ -0,0 +1,44 @@
+from project1_counter.instance_counter import instances_counter
+
+
+@instances_counter
+class User:
+    pass
+
+
+@instances_counter
+class Automobile:
+    color = "black"
```

Comment:
Lets rename `color` to the `instances_amount`

Reply:
A, ok, you probably mean instances_amount in ClassHelper should be private - __instances_amount. Fixed

Reply:
This feature was added to Python exactly for that case.